src/main/java/Duke/DukeInit.java

```diff
@@ -0,0 +1,26 @@
+package Duke;
+
+import Commands.CommandHub;
+import Database.Storage;
+
+/**
+ * Initialises the bot
+ */
+public class DukeInit {
+
+    /**
+     * Initalises bot.
+     */
+    public static void initBot(){
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("I am a very angry bot and I hate you");
+        CommandHub.initCreator(); //init commands
+        Storage.memory_Init(); //init memory
```

Comment:
Interesting use of initialisation methods to start up the classes! Also interesting that you used static methods in those classes, which removes the need to create an instance of the class :)